package.json

```diff
@@ -30,11 +43,14 @@
     "css"
   ],
   "scripts": {
-    "test": "karma start --single-run",
+    "test": "yarn test:build && karma start --single-run",
+    "test:build": "yarn test:build:commonjs && yarn test:build:esm",
+    "test:build:esm": "node lib/esm/index.js",
+    "test:build:commonjs": "node lib/cjs/index.js",
     "tdd": "karma start",
     "check:size": "rollup -c",
-    "build:pick": "cherry-pick --name=dom-helpers --cwd=lib --input-dir=../src --cjs-dir=cjs --esm-dir=esm",
-    "build": "rimraf lib && node_modules/.bin/build src && yarn build:pick",
+    "build": "rimraf lib && node_modules/.bin/build src && yarn build:esm:package.json",
+    "build:esm:package.json": "echo '{ \"type\": \"module\" }' > lib/esm/package.json",
```

Comment:
Having a package.json file with `{ type: module }` is required for Node to treat `lib/esm/*.js` files as ES modules.

Reply:
This isn't a sufficient package.json tho?